_src/m2-sku-reindexing-form.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: m2-documentation
+title: SKU reindexing form
+permalink: /doc/m2/sku-reindexing-form/
+description: Learn about SKU reindexing form in the Algolia extension for Magento 2
+---
+
+## Access
+
+This feature can be accessed directly from the Magento2 back-office main menu :
+**Stores > Algolia Search > Reindex SKU(s)**
+
+## Purpose
+
+It might happens that for some reason, customers want to index one or many products to Algolia immediately, without waiting for the [the indexing queue](/magento/doc/m1/indexing-queue/).
+This tool is also useful to check if your products meets all the requirements to be indexed in the Algolia's indices. 
+
+Keep in mind that if a product can't be reindexed with this tool, it won't be reindexed throught the standard reindexing scripts neither.
+  
+## How to use ?
+
+The form only requires the unique Magento's identifier for all products : the **SKU**.
+
+<img src="../../../img/sku-form.png" class="img-responsive" alt="SKU reindexing form">
+
+To reindex products, you just have to copy/paste their SKUs separated by commas or carriage returns and click **Reindex SKU(s)** on top-right corner of the page.
+
+**Note** : You can reindex up to 10 products at once.
+
+## Understand the outputted results
+
+After submitting the form, you'll get the results. If everything went fine, you'll be notified that the products have been reindexed in all the indices related to the stores of your Magento website.
+**Note** : If the products are not related to some of the stores, they won't be reindexed on those particular indices and you will be notified. 
+
+There are many reasons why a product can't be reindexed :
```

Comment:
Could there be other reasons?

Reply:
No, that's the 4 reasons that are checked during the process